Resources/doc/events.md

```diff
@@ -35,4 +35,4 @@ Its build-in subscribers:
 ## Other listeners 
 
 There is also one subscriber which subscribes all of the above events and is responsible for saving changes to the
-corresponding ORM/ODM - [DoctrineUserListener](EventListener/DoctrineUserListener.php) by the ODM/ORM.
+corresponding ORM/ODM - [PersistDoctrineUserListener](EventListener/PersistDoctrineUserListener.php) by the ODM/ORM.
```

Comment:
"There is also one subscriber which subscribes all of the above events and is" => "There is also a subscriber for all of the above events and it is"